### PR TITLE
Add Whitesource Scans for new Kyma components

### PR DESCRIPTION
**Pull request by Halamix2**
**Description**

Changes proposed in this pull request:

- Add whitesource scans for
    - [kyma-operator](https://github.com/kyma-project/kyma-operator)
    - [kyma-watcher](https://github.com/kyma-project/kyma-watcher)
    - [manifest-operator](https://github.com/kyma-project/manifest-operator)

**Related issue(s)**

- https://github.com/kyma-project/kyma-operator/issues/48

### CODE REVIEW

**Review by Ressetkk — APPROVED** [review]

File: templates/data/whitesource-periodics-data.yaml [hunk 1]
```diff
@@ -100,6 +100,54 @@ templates:
...
+                  global:
+                    - "jobConfig_default"
+                    - "image_wssagent"
+                    - "extra_refs_test-infra"
+                    - "jobConfig_periodic"
+                  local:
+                    - "jobConfig_default"
+                    - "product_kyma"
+              - jobConfig:
+                  repository: "kyma-watcher"
```
Comment (jakobmoellersap) [inline templates/data/whitesource-periodics-data.yaml:120]: I think we should split the watcher scan into skr and kcp as they are 2 go.mod and two internal projects in monorepo style. Is that possible?

**Review by jakobmoellersap — COMMENTED** [review]
Looks great, just a question due to the watcher being 2 go.mods

File: templates/data/whitesource-periodics-data.yaml (hunk 1 above)
Comment (Halamix2) [reply templates/data/whitesource-periodics-data.yaml:120]: resolved in https://github.com/kyma-project/test-infra/pull/5796

**Pull request merged**